automatic/mkvtoolnix/tools/chocolateyInstall.ps1

```diff
@@ -3,18 +3,13 @@
 $packageArgs = @{
   packageName    = 'mkvtoolnix'
   fileType       = 'exe'
-  url            = 'https://mkvtoolnix.download/windows/releases/17.0.0/mkvtoolnix-32-bit-17.0.0-setup.exe'
-  url64bit       = 'https://mkvtoolnix.download/windows/releases/17.0.0/mkvtoolnix-64-bit-17.0.0-setup.exe'
-  checksum       = '6c36699de2b22e50794baa3d91eb8acf5140b80d2f3d7f89ec36f100635db74c'
-  checksum64     = 'd82078d72cb67f15889d912a0c432eefc4cd97d1725948d5a5f6628ebc6156a6'
-  checksumType   = 'sha256'
-  checksumType64 = 'sha256'
+  file           = gi $toolsPath\*-32-*.exe
```

Comment:
here is one reason why I don't like using globbing patterns for this parameters.
Take a look at the latest appveyor build failure:
https://ci.appveyor.com/project/chocolatey/chocolatey-coreteampackages/build/2763-siygwqra#L173
> ERROR: Cannot bind argument to parameter 'file' because it is an empty string.

Reply:
That is one of the reasons I like it :)

Reply:
With the correct package that can't happen anyway.